CHANGELOG.md

```diff
@@ -50,6 +70,30 @@ for downloading the blocks again.
   `iswitness`, `witness_version` and `witness_program`.
   (a28ffa272a3c4d90d0273d9aa223a23becc08e0e)
 
+### Node API changes
+
+#### HTTP
+
+Several CPU and memory exhaustion issues have been resolved with some
+additional arguments for querying multiple sets of results for addresses
+that have many transactions.
```

Comment:
This is missing the update to `GET /`, which adds the `.indexes.{addr,tx}` fields

Reply:
Additions to `/` can be added in the next changelog update, and has been already added to https://github.com/bcoin-org/bcoin-org.github.io/pull/145

Reply:
Added.